packages/docmanager-extension/schema/plugin.json

```diff
@@ -85,6 +89,11 @@
         "command": "docmanager:show-in-file-browser",
         "selector": "[data-type=\"document-title\"]",
         "rank": 23
+      },
+      {
+        "command": "docmanager:duplicate",
+        "selector": "[data-type=\"document-title\"]",
+        "rank": 24
```

Comment:
Should we move the new item to be displayed after the rename action as on the file menu?

Reply:
@fcollonval That would be fine by me, though @krassowski [has noted that the File menu is getting pretty long](https://github.com/jupyterlab/jupyterlab/issues/12099#issuecomment-1207525834). What do you think of removing the option and modifying the behavior of the `Save As...` option? I'm happy to go either way here.

Reply:
:100: Done.

Reply:
@peytondmurray may it be that you forgot to push a commit?